apps/files/src/components/File-Details.vue

```diff
@@ -84,7 +84,7 @@
 		},
 		watch : {
 			file () {
-				OC.$bus.emit('files:file-details-update', this.file)
+				OC.$events.emit('files:file-details-update', this.file)
```

Comment:
do we still need these events ?

Reply:
Haven't touched the files app jet. (Except for some global find and replace stuff)